doc/NEWS.md

```diff
@@ -247,7 +247,7 @@ Thanks to Manuel Mausz for to testing and improving portability!
 - make dbus example work again
 - fix compiler warnings for gcc and clang
 - fix valgrind suppressions
-- Installation of GI binding is fixed, thanks to Dāvis
+- Installation of GI binding is fixed, thanks to Davis
```

Comment:
Is it somehow possible to allow UTF8 by default?
Basically only a latex preamble would be needed, woudn't it?